src/System.Diagnostics.FileVersionInfo/src/System/Diagnostics/FileVersionInfo.cs

```diff
@@ -260,7 +260,7 @@ public string SpecialBuild
         }
 
         /// <summary>
-        /// Returns a System.Windows.Forms.FileVersionInfo representing the version information associated with the specified file.
+        /// Returns a <see cref="T:System.Diagnostics.FileVersionInfo" /> representing the version information associated with the specified file.
```

Comment:
@stephentoub yes, I looked at how other code in the corefx repo did it and it seems like a massive mix of sometimes type prefixed and sometimes full namespaced. I don't have the stats on what was most used, but I saw this syntax used quite a lot so just went with it. Do you want me to change it?